scripts/start.js

```diff
@@ -347,6 +357,10 @@ function getUserInfo () {
     defaultInput: DEFAULT_DOMAIN
   });
 
+  var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
@jniederm , you are right.
The `/api/users` call does not require admin permission. So even if `filtered`, user's id can be retrieved automatically.